keadm/cmd/keadm/app/cmd/beta/config.go

```diff
@@ -169,6 +172,12 @@ func AddImagesCommonConfigFlags(cmd *cobra.Command, cfg *Configuration) {
 	)
 	cmd.Flags().StringVar(&cfg.Part, "part", cfg.Part,
 		"Use this key to set which part keadm will install: cloud part or edge part. If not set, keadm will list/pull all images used by both cloud part and edge part.")
+
+	cmd.Flags().StringVar(&cfg.RuntimeType, cmdcommon.RuntimeType, cfg.RuntimeType,
+		"Container runtime type, default is docker")
+
+	cmd.Flags().StringVar(&cfg.RemoteRuntimeEndpoint, cmdcommon.RemoteRuntimeEndpoint, cfg.RemoteRuntimeEndpoint,
+		"KubeEdge Edge Node RemoteRuntimeEndpoint string")
```

Comment:
s/KubeEdge Edge Node RemoteRuntimeEndpoint string/remoteRuntimeEndpoint is the endpoint of remote runtime service in edge node/g